src/com/messagemedia/clientgui/Manager.java

```diff
@@ -131,7 +136,16 @@ public static String sendMessage(SendPanel panel){
 				validity = Short.parseShort(panel.getValidityPeriodField().getText());
 			}catch(NumberFormatException e){}
 		}
-		
+
+		if (!panel.getScheduledField().getText().equals("")) {
+			try {
+				Date date = DATE_FORMAT.parse(panel.getScheduledField().getText());
+				GregorianCalendar gregorianCalendar = (GregorianCalendar) GregorianCalendar.getInstance();
+				gregorianCalendar.setTime(date);
+				scheduled = DatatypeFactory.newInstance().newXMLGregorianCalendar(gregorianCalendar);
+			} catch (Exception e) {}
```

Comment:
Why should I do this if other pieces of code do not do this? I wanted consistency at least.